ci/utils/test.cue

```diff
@@ -33,6 +33,23 @@ import (
 	}
 }
 
+#TestImageCommand: {
+	testImage:     docker.#Image
+	command:      string
+
+	run: {
+		bash.#Run & {
+			env: {
+				CMD:        "\(command)"
+			}
+			input:  testImage
+			always: true
+			script: contents: #"""
+				composedb "$CMD"
```

Comment:
Just realized that this is now a specifically ComposeDB test action 😅 Possible to pass the full `composedb version` command from the main plan and to just invoke `$CMD` here? That way, we could use this action in other places to run CLI commands.

Reply:
Yeah, I really like the idea. Having some trouble getting it to do that.

I tried making it a list 
https://github.com/3box/pipeline-tools/pull/46/files#diff-a6e49c3d242d80956ca668ff2bb14d79545034eb659c42e489254dbc9a5373e5R42-R49

and I'm getting a command error
```
2:39PM INFO  actions.verify.verifyComposeDB.run._exec | #34 0.110 /bash/scripts/run.sh: line 1: composedb version: command not found
```

Same error if I just left `composedb version` as a string.

Any idea what it's complaining about, or how to idenitify the docker image created or the generated `/bash/scripts/run.sh` file?

Reply:
OK, got it working. Let me know if you don't like this method.